src/app/legacy/components/Promo/index.test.jsx

```diff
@@ -77,3 +77,120 @@ describe('Promo component - Image', () => {
     });
   });
 });
+
+describe('Promo component - Timestamp', () => {
+  describe('Past dates', () => {
+    it('should render timestamp in string format', () => {
+      const stringTimestamp = '2023-11-03T05:17:09.393Z';
+      const { getByText } = render(<Fixture timestamp={stringTimestamp} />, {
+        service: 'mundo',
+      });
+      expect(getByText('3 noviembre 2023')).toBeInTheDocument();
+    });
+    it('should render timestamp in epoch format', () => {
+      const unixTimestamp = 1698995115000;
+      const { getByText } = render(<Fixture timestamp={unixTimestamp} />, {
+        service: 'mundo',
+      });
+      expect(getByText('3 noviembre 2023')).toBeInTheDocument();
+    });
+  });
+
+  describe('Relative times', () => {
+    const epochTimeNow = Date.now();
+
+    const calcTimestampMinutesAgo = minutes =>
+      new Date(epochTimeNow - 60 * 1000 * minutes);
+
+    const calcTimestampHoursAgo = hours =>
+      new Date(epochTimeNow - 60 * 60 * 1000 * hours);
+    describe('Relative time 0 minute ago', () => {
+      it('should render 1 minute ago in string format if time since last published is less than 1 minute ago', () => {
+        // We do not want the timestamp to say 0 minutes if time since last published is less than 1 minute ago.
+        // Relative time stamp will say 1 minute ago for last published between 0 minutes and 1.59 minutes
+        const zeroMinuteAgoString = calcTimestampMinutesAgo(0).toISOString();
+        const { getByText } = render(
+          <Fixture timestamp={zeroMinuteAgoString} />,
+          {
+            service: 'mundo',
+          },
+        );
+        expect(getByText('1 minuto')).toBeInTheDocument();
+      });
+      it('should render 1 minute ago in epoch format if time since last published is less than 1 minute ago', () => {
+        const zeroMinuteAgoEpoch = calcTimestampMinutesAgo(0).getTime();
+        const { getByText } = render(
+          <Fixture timestamp={zeroMinuteAgoEpoch} />,
+          {
+            service: 'mundo',
+          },
+        );
+        expect(getByText('1 minuto')).toBeInTheDocument();
+      });
+    });
+    describe('Relative time 59 minutes ago', () => {
+      it('should render 59 minutes ago or less in string format', () => {
```

Comment:
I don't think we need to include orLess in the test descriptions - let's remove it from all others 👇 too
```suggestion
      it('should render 59 minutes ago in string format', () => {
```

Reply:
Also, just a small nitpick, do we need to repeat the same text in the `describe` and the `it`, as it is kinda repeated / duplicated? Let's simplify the test descriptions (`it`) as much as we can to avoid repeated text.

Reply:
'59 minutes or less' describes the requirement, but here the test is with just '59 minutes', not less, so we can reword this. Requirements can be worked out from looking at the tests anyway and understanding they are at boundaries.